.github/workflows/fuzzing-ci.yaml

```diff
@@ -51,9 +51,13 @@ jobs:
       run: |
         pip install typing boofuzz
 
-    - name: run test
+    - name: run simpleroute test
       run: |
-        python $PWD/t/fuzzing/route_test.py
+        python $PWD/t/fuzzing/simpleroute_test.py
+
+    - name: run serverless route test
```

Comment:
Why put it into separate step?

Reply:
I think this should be clearer. what do you think?

Reply:
I see.